tipg/dbmodel.py

```diff
@@ -507,24 +508,29 @@ def _where(
             wheres.append(to_filter(cql, [p.name for p in self.properties]))
 
         if tile and tms and geometry_column:
-            tilebox = tms.bounds(tile)
+            # Get Tile Bounds in Geographic CRS (usually epsg:4326)
+            left, bottom, right, top = tms.bounds(tile)
```

Comment:
One potential issue is that morecantile do not enforce the geographic CRS to be `epsg:4326`, we should maybe only accept TMS that uses WGS84. (note this is because morecantile is used with non-earth bodies)